instructor/common.py

```diff
@@ -459,24 +477,23 @@ def strain_treatments_edit(request):
             headers.append(optional_vars[index])
 
     STFormSet = modelformset_factory(models.StrainTreatment, extra=0,
-                                     exclude=['assignment', 'strain', 'treatment', 'enabled'],
-                                     can_delete=True)
+                                     exclude=['assignment', 'strain', 'treatment'])
     if request.method == "POST":
         formset = STFormSet(request.POST)
         formset.clean()
         if formset.is_valid():
             entries = formset.save(commit=False)
-            for obj in formset.deleted_objects:
-                obj.delete()
             for form in entries:
                 form.save()
-    else:
-        # If there are no Treatments want to create them
-        if not models.Treatment.objects.filter(assignment=assignment).exists():
-            create_treatments(assignment)
-            create_strain_treatments(assignment)
 
-    formset = STFormSet(queryset=models.StrainTreatment.objects.filter(assignment=assignment))
+    formset = STFormSet(
+        queryset=models.StrainTreatment.objects.filter(assignment=assignment).order_by(
+            'strain',
+            'treatment__drug__name',
+            'treatment__temperature__degrees',
```

Comment:
Since these fields are text this will be ordered in alphabetical order, just FYI. It's fine as is